### DISCUSSION TITLE
Welcome to the rest.guide discussion!

### DISCUSSION THREAD

**Discussion by malcolmsparks** [juxt/rest#1] [General]
## Welcome!

We’re using GitHub's Discussions as a place to talk about content on rest.guide. Please feel free to ask any questions, share feedback and ideas, suggest changes and post code snippets.

For this discussion, why not comment below to introduce yourself?

**Comment by evaogbe** [comment]
Hi I'm Eva. I'm interested in building Clojure applications/libraries. I want to learn more about REST. Is this discussion page/repo still active? I feel a little weird commenting on something 2 years old, but I figured it wouldn't hurt to reach out.

**Comment by malcolmsparks** [comment]
Hi Eva - sorry for the delay in responding. All my activity these days is over at https://github.com/juxt-site where we've been building on the ideas in rest.guide. 

As for learning REST, I still feel one of the best starting points is Roy Fielding's original dissertation (https://ics.uci.edu/~fielding/pubs/dissertation/top.htm). There a good summary here: https://oleb.net/2018/rest/

I'd like to put together a blog or document teaching REST principles by building a simple set of endpoints in Site, since Site aims to provide a complete 100% conformant implementation of HTTP with REST principles.
